mmv1/third_party/terraform/go.mod.erb

```diff
@@ -4,6 +4,7 @@ module github.com/hashicorp/terraform-provider-google<%= "-" + version unless ve
 go 1.18
 
 require (
+	cloud.google.com/go/alloydb v0.2.0
```

Comment:
Can we avoid adding new GRPC clients? Let's use https://github.com/googleapis/google-api-go-client or make HTTP calls directly similar to MMv1 instead.

Reply:
Sent https://github.com/GoogleCloudPlatform/magic-modules/pull/7544 to update guidance! Sorry, we hadn't considered folks would use a veneer client.